docs/throttling.md

```diff
@@ -50,7 +50,7 @@ Currently, `comcast` will also throttle the websocket port that Lighthouse uses
 comcast --latency=150 --target-bw=1600
 
 # Run Lighthouse with it's own throttling disabled
```

Comment:
will you fix the `it's` while youre in here?

Reply:
haha done